node/src/actors/peers_manager/mod.rs

```diff
@@ -141,6 +141,24 @@ impl PeersManager {
         });
     }
 
+    /// Method to periodically melt peers
+    fn melt_peers(&self, ctx: &mut Context<Self>, check_melted_peers_period: Duration) {
+        // Schedule the melt_peers with a given period
+        ctx.run_later(check_melted_peers_period, move |act, ctx| {
+            // Remove peers from `ice` bucket that have "melted", i.e. they have been in that bucket
+            // for as long as specified by the `connections.bucketing_ice_period_seconds` setting in
+            // the configuration. These peers will be added to `new` again, so as to ensure that
+            // they never get stuck in the `ice` bucket
+            let addresses = act.peers.extract_melted_peers_from_ice_bucket();
+            if !addresses.is_empty() {
+                log::debug!("Melt these addresses: {:?}", addresses);
```

Comment:
One of these is maybe nicer:
```suggestion
                log::debug!("Melted these addresses: {:?}", addresses);
```
```suggestion
                log::debug!("Melting these addresses: {:?}", addresses);
```